### DISCUSSION TITLE
Thinking to rename Ebiten to a new name due to the trademark issue

### DISCUSSION THREAD

**Discussion by hajimehoshi** [hajimehoshi/ebiten#1500] [General]
Hi all, this is not an urgent issue but I'm thinking to rename Ebiten to something due to trademarks in Japan. Ebiten (Ebten) is already registered by a company in Japan.

There are some candidates:
* Tendon (Tempura and rice bowl)
* Oden (Japanese one-pot dishes) (EDIT: Apparently this was already registered by oden.io)
* Shrimp
* Nebiten (небитен: being unimportant in Macedonian?)
* Ebeaten 

We have not decided anything. Perhaps, we might decide to change nothing (and ignore the trademark issue).

I'd like to hear your opinions. Thanks,

EDIT: Probably nothing changes within 4 or 5 years. This is a very long-span issue.

**Comment by griffithsh** [comment]
*Tendon* has an unfortunate (or maybe it's fortunate?) overloading in English.

https://www.google.com/search?q=define+tendon

> **tendon**
> /ˈtɛndən/
> *noun*
> a flexible but inelastic cord of strong fibrous collagen tissue attaching a muscle to a bone.
> the hamstring of a quadruped.

**Reply by hajimehoshi** [reply]
Yeah, that's a coincidence, but I think this is not bad.

**Reply by dbriemann** [reply]
For what it's worth, my 5 cent. I would avoid overloaded names if possible, at least overloading with the English language.
If you do something like that it should be especially funny or clever. 

In this case the English meaning of tendon does not seem negative or positive but it just adds confusion. People will probably not get why the library is named after _a flexible but inelastic cord of strong fibrous collagen tissue attaching a muscle to a bone_.

**Comment by eihigh** [comment]
This site might be useful. https://experience-japan.info/

I found tofu, udon, soba, natto, senbei, anpan, ...

**Reply by hajimehoshi** [reply]
Yeah, I'd like to use Japanese food names. Thanks!

**Comment by magnuswahlstrand** [comment]
Oden is how we spell the Norse god Odin. Might be why the name is taken 🤔

Shrimp is pretty cool 🦐🍤

**Reply by hajimehoshi** [reply]
Thank you! Yeah, I'd like to keep the theme of shrimps.

**Reply by loig** [reply]
If so, maybe finding a language in which the equivalent for shrimp sounds/looks cool to you? In french, for example, it is called "crevette".

**Reply by jmaister** [reply]
shrimpura - form shrimp tempura, so it will not collide with real food names in search engines

**Comment by dcoty** [comment]
If you want to keep with the crustacean theme, how about Botanebi? Or maybe Prawn, but i think that might be taken. I'll think of some more soon :)

**Reply by hajimehoshi** [reply]
Thank you! Botanebi sounds yummy!

**Comment by TheMightyGit** [comment]
Perhaps add something to it rather than totally change it? EbitenLib EbitenGo EbitenGame Ebiten+[something]

**Reply by hajimehoshi** [reply]
(An existing trademark) + alpha might be problematic, but sounds nice!

**Comment by gabstv** [comment]
I don't know how Japan's trademark laws work, but usually you could keep the same phonetics, but changing/adding/removing vowels or consonants?

**Reply by hajimehoshi** [reply]
It depends. For example, 'Ebitempura' might not be safe since this is the same meaning of 'Ebiten'. 'Ebitendon' might be safe since this is a little different meaning.

**Comment by timbear1** [comment]
I don't have any comments on the actual new name, but maybe favor something that is short-ish as it's going to be written a lot.

**Reply by hajimehoshi** [reply]
Yeah, I'll try to make it short!

**Comment by Nassim-Saboundji** [comment]
How about Ebi10 ? With 10 pronounced in english you keep the same pronunciation . Or you could change it to Ebi9 or another any number you want.

**Reply by hajimehoshi** [reply]
The same pronunciation as an existing trademark cannot be registered in Japan. Ebi11 or Ebi9 sounds unique :)

**Comment by harbdog** [comment]
How about EbiEbi? So nice you can name it twice :)

**Reply by hajimehoshi** [reply]
Not bad. I've confirmed EbiEbi is not registered.

**Comment by ghtalpo** [comment]
How about libebi? Looks more game library. :)

**Reply by hajimehoshi** [reply]
Thank you for the suggestion!

**Comment by occamist** [comment]
Pretty dumb idea but how about netibe? The reverse of existing name.

There is also popular name called "Nevide" which means exciting, cheerful and great news in arabic. Reverse of it would be "ediven" which is kinda close to ebiten but softer to pronounce. Kinda reminds me aiven kafka provider managed service based in Finland lol

**Reply by hajimehoshi** [reply]
Reversing the spelling sounds an interesting idea, thanks!

**Comment by hajimehoshi** [comment]
Ebitenorg (just to keep the domain ebiten.org)

**Comment by alejandro-a** [comment]
What about Teriyaki or Ebiyaki?

**Reply by hajimehoshi** [reply]
Thanks! Teriyaki is interesting. Unfortunately there is no word Ebiyaki.

**Comment by shaneutt** [comment]
I think Oden is great because it is short and simple. I see that the .io domain is taken so maybe you're counting that one out of the running but so far that's my favorite.

**Reply by hajimehoshi** [reply]
Though Oden is a fancy name, this is already registered (maybe as a software). I'll double-check.

**Reply by hajimehoshi** [reply]
https://branddb.wipo.int/branddb/en/showData.jsp?ID=USTM.86577190

**Comment by eihigh** [comment]
How about `gamite` (gay-my-t, ゲーマイト in Japanese) ? In this word, `-ite` means "rough stone" with great potential. And it's a word that doesn't even exist in the world yet!

**Reply by hajimehoshi** [reply]
Thanks! I have confirmed this is not used or registered anywhere. One concern is that this doesn't keep the original Ebiten meaning...

**Comment by hajimehoshi** [comment]
Finally I have decided to use a new name "Ebitengine" (pronounced as Ebi-Ten-Jin).

Thank you for your nice suggestions. Every suggestion was actually really nice, and I'd like to express my thanks to all of you.

**Reply by occamist** [reply]
engine??, are you guys sure :D people will say it like a bit engine :D

**Reply by hajimehoshi** [reply]
Yes, it's an engine!

**Reply by occamist** [reply]
ok then there are 2 pronunciations "a bit engine" (I like this one) and "ebi ten jin", it is much like "godot" being "go dough" and "go dot" (I still use this one) :D
